.github/contributing/components.md

```diff
@@ -188,18 +177,15 @@ must be written in `camelCase` and follows the rules written in [Base-naming](ba
 
 ### `<event>:<source>`
 
-If a component can emit multiple types of events, like clicks depending on which subcomponent has
-been clicked, for instance, name the events following the `<event>:<source>` convention in kebab
-case. The `event` part is the name of the DOM event that triggered the Vue event, and the `source`
-one the name of the element that triggered the event.
+If a component can emit multiple types of events, such as clicks depending on the subcomponent clicked, you must name the events according to the `<event>:<source>` convention in `kebab` case. 
+`event` represents the name of the DOM event that triggered the Vue event, and `source` represents the name of the element that triggered the event.
 
 - `click:close`, `click:item`, `click:link`, `click:right-button`.
 
 ## Naming getters
 
-Getters belong to x-modules. There are two different ways of declare a getter. It can be a function
-or a class with multiple getters. If it is a function, it should be named using `camelCase` and
-should explain what it is returning.
+Getters belong to [X Module](./concepts.md#x-module). There are two different ways of declare a getter. It can be a **function** or a **class** with multiple getters. 
```

Comment:
```suggestion
Getters belong to [X Module](./concepts.md#x-module). There are two different ways of declareing a getter. It can be a **function** or a **class** with multiple getters. 
```

- [ ] Fix `declare` typo